tests/test_configuration.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: copy load-balancer configuration fragment to the server
+  copy:
+    content: "{{ config_fragment }}"
+    dest: /tmp/config-fragment
+- name: copy load-balancer backend map fragment to the server
+  copy:
+    content: "{{ backend_map_fragment }}"
+    dest: /tmp/backend-map-fragment
+- name: apply the new configuration
+  command: haproxy-config apply test /tmp/config-fragment /tmp/backend-map-fragment
+- name: wait for the certificate to be available
```

Comment:
My integration tests are timing out at this step.  It looks like the .pem file gets created fine, but it doesn't have the expected file name?
```
root@jill-test-haproxy:/etc/haproxy/certs# ll 
total 20
drwx------ 2 root root 4096 Nov 17 05:38 ./
drwxr-xr-x 6 root root 4096 Nov 17 05:05 ../
-rw-r--r-- 1 root root 5176 Nov 17 05:38 jill-test-haproxy.plebia.net-0001.pem
-rw-r--r-- 1 root root 2729 Nov 17 05:52 ssl-cert-snakeoil.pem
```

If I copy `jill-test-haproxy.plebia.net-0001.pem` to `jill-test-haproxy.plebia.net.pem` while ansible is waiting for the file to appear, the tests continue, but fail here:

```
TASK [verify that the backend is accessible via HTTPS] *************************
task path: /home/jill/instance-manager/ansible-load-balancer/tests/test_configuration.yml:22
ESTABLISH LOCAL CONNECTION FOR USER: jill
localhost EXEC /bin/sh -c '( umask 22 && mkdir -p "` echo $HOME/.ansible/tmp/ansible-tmp-1479362762.39-251985061172904 `" && echo "` echo $HOME/.ansible/tmp/ansible-tmp-1479362762.39-251985061172904 `" )'
localhost PUT /tmp/tmpClgkG2 TO /home/jill/.ansible/tmp/ansible-tmp-1479362762.39-251985061172904/command
localhost EXEC /bin/sh -c 'LANG=en_AU.UTF-8 LC_ALL=en_AU.UTF-8 LC_MESSAGES=en_AU.UTF-8 /data/jill/.virtualenv/ansible-load-balancer-p2/bin/python2 /home/jill/.ansible/tmp/ansible-tmp-1479362762.39-251985061172904/command; rm -rf "/home/jill/.ansible/tmp/ansible-tmp-1479362762.39-251985061172904/" > /dev/null 2>&1'
fatal: [jill-test-haproxy.plebia.net -> localhost]: FAILED! => {"changed": true, "cmd": ["wget", "-O-", "--ca-certificate=../files/lets-encrypt-fake-cert.crt", "https://jill-test-haproxy.plebia.net/"], "delta": "0:00:05.140661", "end": "2016-11-17 16:36:07.617502", "failed": true, "invocation": {"module_args": {"_raw_params": "wget -O- --ca-certificate=../files/lets-encrypt-fake-cert.crt \"https://jill-test-haproxy.plebia.net/\"", "_uses_shell": false, "chdir": null, "creates": null, "executable": null, "removes": null, "warn": true}, "module_name": "command"}, "rc": 8, "start": "2016-11-17 16:36:02.476841", "stderr": "--2016-11-17 16:36:02--  https://jill-test-haproxy.plebia.net/\nResolving jill-test-haproxy.plebia.net (jill-test-haproxy.plebia.net)... 213.32.78.69\nConnecting to jill-test-haproxy.plebia.net (jill-test-haproxy.plebia.net)|213.32.78.69|:443... connected.\nHTTP request sent, awaiting response... 503 Service Unavailable\n2016-11-17 16:36:07 ERROR 503: Service Unavailable.", "stdout": "", "stdout_lines": [], "warnings": ["Consider using get_url module rather than running wget"]}
```

Note that this is my second time through running the integration tests on my server.  The first time, I ran them using the `origin/master` tests.  Do I have to create a fresh server each time or something?